PRINCIPLES.md

```diff
@@ -10,22 +10,22 @@ These principles were derived from modern software operations but are rooted in
 
 ## Principles
 
-1. **The principle of declarative desired state**
+1. **Desired System State Is Declarative**
 
     A system managed by GitOps must have its _Desired State_ expressed declaratively as data in a format writable and readable by both humans and machines.
 
-2. **The principle of immutable desired state versions**
+2. **Declarations Are Versioned and Immutable**
 
     _Desired State_ is stored in a way that supports versioning, immutability of versions, and retains a complete version history.
 
-3. **The principle of continuous state reconciliation**
+3. **State Reconciliation Is Continuous**
```

Comment:
```suggestion
3. **State Reconciliation is Continuous**
```
Typographic Suggestion:
`Is` -> `is`
to align with
https://user-images.githubusercontent.com/407675/126000898-eb81abd1-a947-40e4-b313-107fd45b7ad7.png
Thanks @scottrigby

Reply:
See https://github.com/open-gitops/documents/pull/14#discussion_r676641157